db/migrate/20230607161527_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counters
+      t.integer :author_id
```

Comment:
- [ ] Take a look at the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). You should create the associations or foreign keys column using [references](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) column type
also available as `belongs_to`. Your code should look similar to this:

```ruby
  create_table :posts do |t|
    t.string :title
    t.text :text
    ...
    t.references :author, foreign_key: { to_table: 'users' }
  end
```
- This will create author_id and a corresponding index.